services/cubejs/index.js

```diff
@@ -217,13 +216,34 @@ const options = {
       dataSchemaFiles: () => dataSchemaFiles({ dataSourceId, authToken }),
     };
   },
+  preAggregationsSchema: ({ securityContext }) => `pre_aggregations_${securityContext?.dataSourceVersion}`,
   telemetry: false,
```

Comment:
there's an unused `CUBEJS_TELEMETRY` env var. Maybe it's better to use here?

Reply:
done